tools/ExtensionLibrary/DevHome.ExtensionLibrary/Strings/en-us/Resources.resw

```diff
@@ -134,4 +193,13 @@
     <value>More options</value>
     <comment>ToolTip of the button that brings up the "More options" menu</comment>
   </data>
+  <data name="InstalledItemsStackPanel.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Installed</value>
+  </data>
+  <data name="AvailableFromStoreStackPanel.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Available in the Microsoft Store</value>
+  </data>

```

Comment:
Do these actually get read or is it the TextBlocks inside?

Reply:
The `InstalledItemsStackPanel.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name` gets read as the group name (e.g. "Installed group"). If scan mode is on, it would read "Installed group; [and if you navigate to the heading] Heading level 2: Installed" (the TextBlock). If the narrator read the "AvailableFromStore" group, it would read its StackPanel's AutomationProperties.Name. If you navigate to the "Available in the Microsoft Store" heading in scan mode, it reads "Heading level 2: "Available in the Microsoft Store" (the TextBlock).

I think the narrator reads the StackPanel Automation Properties.Name as the group name because the item groups underneath them are [ItemsRepeaters](https://learn.microsoft.com/en-us/windows/apps/design/controls/items-repeater#enable-accessibility), not Lists or another control with a designated header.